haskell/protobuf.bzl

```diff
@@ -206,8 +207,20 @@ def _haskell_proto_aspect_impl(target, ctx):
             transitive_html.update(dep[HaddockInfo].transitive_html)
             transitive_haddocks.update(dep[HaddockInfo].transitive_haddocks)
 
+    package_id = library_info.package_id
+
+    # TODO
+    # Missing haddock information for this build
+    # See bug https://github.com/tweag/rules_haskell/issues/1030
+    # We instead declare empty documentation directories / file
+    haddock_files = []
+    html_dir = None
+
+    #transitive_html.update({package_id: html_dir})
+    transitive_haddocks.update({package_id: haddock_files})
+
     haddock_info = HaddockInfo(
-        package_id = None,
+        package_id = package_id,
```

Comment:
Is it necessary to duplicate `package_id` in `HaddockInfo`, or would the field in `HaskellLibraryInfo` suffice? In other words, do we have targets that provide `HaddockInfo` but not `HaskellLibraryInfo`?

Reply:
I don't believe we need this duplication.

Reply:
I do agree with you. However I thank that's orthogonal to this pull request.

Reply:
Fair enough. That can also be done in a separate PR.